colossalai/inference/core/request_handler.py

```diff
@@ -105,6 +108,11 @@ def schedule(self):
                             )
                             self.abort_sequence(seq.request_id)
                             break
+
+                        # stop feeding new sequence into running list to assure
+                        if self.cache_manager.num_available_blocks <= self.running_list.total_seq_num:
```

Comment:
What does this mean?

Reply:
make sure that we have `available  blocks ` more than sequences in the running list, so we can at least do a batch step

Reply:
OK